### PR TITLE
change hof-bootstrap to MIT license..

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
* remove licensing of the example-app as this should refer to the hof-bootstrap license
* add .idea to .gitignore to ignore IntelliJ files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "hof-bootstrap",
   "description": "A bootstrap for HOF projects",
   "version": "6.0.1",
-  "license": "GPL-3.0",
+  "license": "MIT",
```
Comment (joefitter) [inline package.json]: Don't we need to add an actual text copy of the license?

**Comment by joefitter** [comment]
@sulthan-ahmed the LICENSE file in this repo is   GNU GENERAL PUBLIC LICENSE

**Comment by sulthan-ahmed** [comment]
Ahh right 👍  I didn't know that's how you put licenses, I thought it was in the package.json.  I'll make the change

**Comment by sulthan-ahmed** [comment]
made the change @joefitter

**Comment by joefitter** [comment]
🏩

**Comment by sulthan-ahmed** [comment]
http://www.kappit.com/img/pics/201603_1355_bidea_sm.jpg

**Pull request merged**